### PR TITLE
First HTML Assignment

### PR DESCRIPTION
**Pull request by ayushiagrawal2000**
First HTML Assignment

### CODE REVIEW

File: Assignment 1-HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    <div class="video">
+        <p>How ActivityHero works(in 90 seconds)</p>
+        <iframe src="https://www.youtube.com/embed/srn5jgr9TZo" title="YouTube video player"
+         frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+         allowfullscreen></iframe>
+    </div>
+    <div class="feature">
+        <h3>What You'll Get With ActivityHero</h3>
+        <div class="feature-content">
+            <img src="images/easy-setup.png" alt=""><br>
```
Comment (pranav96vaidya) [inline Assignment 1-HTML/index.html:68]: remove br tag

Comment (ayushiagrawal2000) [reply Assignment 1-HTML/index.html]: fixed it

File: Assignment 1-HTML/index.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            marketplace, serving more than 2.5 million nativwide. Your activities can be include in:
+            <ul>
+                <li>Local city home pages</li>
+                <li>Activity search result</li>
+                <li>Email newsletters to local families</li>
+                <p>see a <a href="https://newput.slack.com/archives/D021H17PCD8/p1621329237001000" target="_blank">list of cities we server</a></p>
+                <li style="font-size: 13px;">There is no charge tobe promoted, aditional registration fees will be applied on  final registration so be ready.</li>
+            </ul> 
+        </p>
+        </div>
```
Comment (pranav96vaidya) [inline Assignment 1-HTML/index.html:145]: check indentation

File: Assignment 1-HTML/index.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    </div>
+    <div class="registration-page">
+        <div class="register-heading">
+            <h3>Get your first month free!</h3>
+            <p>fill out the form bellow</p>
+        </div>
+            <div class="form-container">
```
Comment (pranav96vaidya) [inline Assignment 1-HTML/index.html:152]: Fix indentation here too

File: Assignment 1-HTML/index.html [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                    <input type="text">
+                    <label for="Phone Number">Phone Number</label>
+                    <input type="text">
+                    <label for="Name">Where do you want to start <br> accepting registration</label>
+                    <input type="text">
+                    <button class="button">Tell me more</button>
+                </form>
+            </div>      
+    </div>
+</body>
```
Comment (pranav96vaidya) [inline Assignment 1-HTML/index.html:168]: Fix indentation here too.

File: Assignment 1-HTML/style.css [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+.button{
+    padding: 7px;
+    width: 170px;
+    background-color: rgb(16, 182, 16);
+    color: white;
+    border: none;
+    font-size: large;
+    cursor: pointer;
+}
+.main-content i{
```
Comment (pranav96vaidya) [inline Assignment 1-HTML/style.css:52]: Add a space between name and opening braces everywhere.

**Pull request merged**